docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -404,3 +404,106 @@ Example output:
 ```text
 SpecVersion=2.0
 ```
+
+## TPM003.001 Check TPM Physical Presence Interface (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that the TPM Physical Presence Interface is correctly
+recognized by the operating system.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+2. `OPERATING_SYSTEM` = Ubuntu 22.04
+3. Platform with TPM module present.
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+2. Boot into the system.
+3. Log into the system by using the proper login and password.
+4. Check the version of TPM PPI ins sysfs:
```

Comment:
@miczyg1 this not the fault, but if we want to keep the used nomenclature, we should write the fourth point in the following manner:

4. Open the terminal and run the following command to check the version of TPM PPI ins sys:

```
cat /sys/class/tpm/tpm0/ppi/version
```

Reply:
Sure, being consistent is important. Fixed @sulewskiprzemyslaw